content/posts/blog/easy-xmpp-challenge.md

```diff
@@ -87,12 +86,39 @@ are a first step in that direction, and getting them into your client today
 will make new users' lives easier. But there is also potential to further
 streamline the whole process.
 
-### Call for Action
+### Easy Group Chats
+
+There are two typical IM use cases for group chats: chat with multiple friends
+at the same time (ad-hoc), or participate in a public chat room (typically for
+support purposes, pioneered by
+[Internet Relay Chat](https://en.wikipedia.org/wiki/Internet_Relay_Chat)).
```

Comment:
There is a third one: Have a private group, e.g. for your family. That’s not really ad-hoc, because you want it to be as persistent as possible. Maybe my understanding of Ad-Hoc is off though. At least this could use some clarification along the lines of "(ad-hoc, which doesn’t necessarily mean that it’s not persistent)".